sync/src/block/downloader/body.rs

```diff
@@ -83,6 +83,12 @@ impl BodyDownloader {
         }
     }
 
+    pub fn reset_downloading(&mut self, hashes: Vec<H256>) {
```

Comment:
No need to move `hashes`.
How about `fn reset_downloading(&mut self, hashes: &[H256])`?

Reply:
I agree, and also other functions in `BodyDownloader` can be changed in that way.

Reply:
I'll fix `reset_downloading` and `remove_target`.

Reply:
Fixed `reset_downloading`. Others may be fixed in the next.